tutor/src/screens/assignment-review/ux.js

```diff
@@ -239,7 +239,7 @@ export default class AssignmentReviewUX {
 
     await this.editUX.initialize({
       ...this.params,
-      plan: this.taskPlan,
+      plan: cloneDeep(this.taskPlan),
       history,
```

Comment:
This doesn't seem like it should be needed.  assignment-edit/ux already creates a new task plan in order to prevent this sort of thing. Otherwise the changes would be saved to the in-memory plan even if the user hit cancel.  If there is leakage of objects, we should fix it there IMO

https://github.com/openstax/tutor-js/blob/wrm/tutor/src/screens/assignment-edit/ux.js#L59-L63

Reply:
Thanks for this, didn't realize we were doing that in the edit UX and that we needed to pass the plan id here instead of the object.